### DISCUSSION TITLE
How to change my own status

### DISCUSSION THREAD

**Discussion by ToolifyStudios** [stoatchat/discussions#515] [Q&A] [answered]
I'm new here to Revolt, it's pretty cool
I just don't know how to change my status

Can anyone help me please?

**Comment by Rexogamer** [comment, marked as answer]
Right click (or, on mobile, long tap) your user icon on the top right corner:

https://user-images.githubusercontent.com/42586271/225781997-207ba1a2-a011-4a93-8edf-1a7d1e476d69.MOV
